integration_tests/src/main/python/cmp_test.py

```diff
@@ -57,7 +57,9 @@ def test_ne(data_gen):
                 f.col('b') != f.lit(None).cast(data_type),
                 f.col('a') != f.col('b')))
 
-@pytest.mark.parametrize('data_gen', orderable_gens, ids=idfn)
+@incompat
+@pytest.mark.allow_non_gpu('ProjectExec', 'Alias')
```

Comment:
These should be changed now.  It is not incompat and Project and Alias are on the GPU in all cases.

Reply:
Done